pydra/engine/tests/test_helpers_file.py

```diff
@@ -184,6 +186,79 @@ def test_get_related_files_noninclusive(_temp_analyze_files):
     assert orig_hdr not in related_files
 
 
+def test_hash_file_broken(_temp_analyze_files):
+    """
+    Test how broken file paths are handled during hashing
+
+    When `raise_notfound` is True, raise error for broken file paths or
+    broken symlinks. Otherwise return None
+    """
+    # broken file path
+    assert hash_file("broken_file.txt", raise_notfound=False) is None
+    with pytest.raises(FileNotFoundError) as e1:
+        hash_file("broken_file.txt", raise_notfound=True)
+    assert "File doesn't exist" in str(e1)
```

Comment:
not completely sure if this is the only problem with failing tests on travis, but you should probably use `str(e1.value)` in all places you are checking the error message

Reply:
Thanks for the suggestion!